### PR TITLE
Add AccountProfile

### PR DESCRIPTION
**Pull request by NikolasStelmakh**

### CODE REVIEW

File: server/db/migrations/20220411191359-add_account_profile_table_and_refactor_wallet.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+          profile_name: {
+            type: DT.STRING,
+            allowNull: true
+          },
+          is_default: {
+            type: DT.BOOLEAN,
+            allowNull: false,
+            defaultValue: false
+          },
+          created: {
```
Comment (andreyvEze) [inline server/db/migrations/20220411191359-add_account_profile_table_and_refactor_wallet.js:45]: there are predefined props for it = `created_at`, could we use it?

File: server/db/migrations/20220411191359-add_account_profile_table_and_refactor_wallet.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+          allowNull: false,
+          defaultValue: ''
+      });
+      await QI.addColumn('wallet', 'permission', {
+          type: DT.STRING,
+          allowNull: false,
+          defaultValue: ''
+      });
+      await QI.removeColumn('wallet', 'account_profile_id');
+      return QI.dropTable('actor');
```
Comment (andreyvEze) [inline server/db/migrations/20220411191359-add_account_profile_table_and_refactor_wallet.js:90]: could we add queries to fill previous data?

File: server/db/migrations/20220411191359-add_account_profile_table_and_refactor_wallet.js [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+          },
+          actor: {
+            type: DT.STRING,
+            allowNull: false,
+          },
+          permission: {
+             type: DT.STRING,
+             allowNull: false
+          },
+          profile_name: {
```
Comment (andreyvEze) [inline server/db/migrations/20220411191359-add_account_profile_table_and_refactor_wallet.js:36]: could we rename to `name`?

File: server/routes/admin.js [hunk 4]
```diff
@@ -844,4 +852,133 @@ router.get('/csv-report', handler(async (req, res) => {
...
+  })
+
+  return res.send({list: accProfiles.map(acc => acc.get())})
+}))
+
+router.delete('/account-profile/:id', handler(async (req, res) => {
+  const {user_id} = res.state
+  if(!user_id) {
+    return res.status(401).send({error: 'Unauthorized'})
+  }
```
Comment (andreyvEze) [inline server/routes/admin.js:877]: could we create one method for it

File: server/routes/admin.js [hunk 5]
```diff
@@ -844,4 +852,133 @@ router.get('/csv-report', handler(async (req, res) => {
...
+
+  const deleteId = req.params.id
+  assert(deleteId !== undefined, 'required id (account_profile id)')
+
+  const rows = await db.AccountProfile.destroy({where: {id: deleteId, is_default: false}})
+
```
Comment (andreyvEze) [inline server/routes/admin.js:883]: should we also set default for all wallets which using it

File: src/pages/Wallet.vue [hunk 6]
```diff
@@ -134,42 +134,20 @@
...
-        <!--label="Domains:" label-for="domains"-->
-      <!--&gt;-->
-        <!--<b-form-textarea id="domains" v-model="form.domains"-->
-          <!--placeholder="Enter domains" required-->
-          <!--:state="domainsValidation"-->
-        <!--&gt;-->
-        <!--</b-form-textarea>-->
-
-        <!--<small class="text-muted">Provide each @<b>domain</b> on a separate line (omit the preceding at symbol).  These are public domains that will be selling addresses on this server.</small>-->
-      <!--</b-form-group>-->
```
Comment (andreyvEze) [inline src/pages/Wallet.vue:171]: should it be removed?

**Review by andreyvEze — CHANGES_REQUESTED** [review]
check comments

File: server/db/migrations/20220411191359-add_account_profile_table_and_refactor_wallet.js (hunk 1 above)
Comment (NikolasStelmakh) [reply server/db/migrations/20220411191359-add_account_profile_table_and_refactor_wallet.js:45]: add created_at, deleted_at (paranoid, timestamps: true) into migration and model

**Review by andreyvEze — APPROVED** [review]

**Review by ericbutz — APPROVED** [review]

**Pull request merged**